app/models/manageiq/providers/redhat/builder.rb

```diff
@@ -0,0 +1,48 @@
+class ManageIQ::Providers::Redhat::Builder
+  class << self
+    def build_inventory(ems, target)
+      case target
+      when ManageIQ::Providers::Redhat::InfraManager
```

Comment:
the switch can be replace by a simple if-else statement:


`
if target == ManageIQ::Providers::Redhat::InfraManager
    inventory(...)
 else
    infra_manager_inventory(...)
end
`